scripts/ide-run-scss-lint-from-docker.sh

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env bash
-
-# This script will run scss-lint
-
-PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)
-
-docker run --rm \
-  -v $PROJECT_DIR:/style-guide \
-  brainly/style-guide bash -c "scss-lint /style-guide/$1 -f JSON | sed 's/\/style-guide\///g'"
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source ${SCRIPT_DIR}/run-scss-lint.sh $1 -f JSON
```

Comment:
This file is used as scss-lint exec file for WebStorm (and all others form the "family"). I can't pass in config of the plugin options to exec so I created proxy file to do it.  Maybe better name would be `ide-scss-lint-exec.sh` ?